eframe/examples/global_font_style.rs

```diff
@@ -0,0 +1,73 @@
+#![cfg_attr(not(debug_assertions), windows_subsystem = "windows")] // hide console window on Windows in release
+
+use eframe::egui;
+use egui::{FontFamily, FontId, RichText, TextStyle};
+
+#[inline]
+fn heading2() -> TextStyle {
+    TextStyle::Name("Heading2".into())
+}
+
+#[inline]
+fn heading3() -> TextStyle {
+    TextStyle::Name("ContextHeading".into())
+}
+
+fn configure_text_styles(ctx: &egui::Context) {
+    use FontFamily::Proportional;
+    use TextStyle::*;
+
+    let mut style = (*ctx.style()).clone();
+    style.text_styles = [
+        (Heading, FontId::new(30.0, Proportional)),
+        (heading2(), FontId::new(25.0, Proportional)),
+        (heading3(), FontId::new(23.0, Proportional)),
+        (Body, FontId::new(18.0, Proportional)),
+        (Monospace, FontId::new(14.0, Proportional)),
+        (Button, FontId::new(14.0, Proportional)),
+        (Small, FontId::new(10.0, Proportional)),
+    ]
+    .into();
+    ctx.set_style(style);
+}
+
+fn content(ui: &mut egui::Ui) {
+    // NOTE: @emilk why the color is different than custom ones?
```

Comment:
because of the call to `strong()`?

Reply:
Yes indeed. removing it made the style consistent